eslint/babel-eslint-parser/test/index.js

```diff
@@ -1,19 +1,43 @@
-import assert from "assert";
 import espree from "espree";
 import escope from "eslint-scope";
 import unpad from "dedent";
 import { parseForESLint } from "../src";
-import assertImplementsAST from "./helpers/assert-implements-ast";
 
-const babelOptions = {
+const BABEL_OPTIONS = {
   configFile: require.resolve(
     "@babel/eslint-shared-fixtures/config/babel.config.js",
   ),
 };
+const ALLOWED_PROPERTIES = [
+  "importKind",
```

Comment:
These properties seem useful for Flow - should we keep them for `eslint-plugin-flow`?